### ISSUE TITLE
background on Boards

### ISSUE THREAD

**Issue by woollensocks** [fguillot/kanboard#695]
Hey,

So have been using Kanboard for a while now and do enjoy it. 

I like backgrounds on my boards so I did it by editing  ..\Kanboard\app\Template\board\show.php to create a new tableid per task for css customization like so : 

<?php if (isset($not_editable)): ?>
        <table id="board"
		class="project<?= $project['id']?>"
		>
    <?php else: ?>
        <table id="board"
		class="project<?= $project['id']?>"
               data-project-id="<?= $project['id'] ?>"

And then simply in Kanboard\assets\css\app.css I put the folowing definations:

table.project1
{
	background-image: url(../img/manx.jpg), url(../img/manx.jpg);
	background-size: 100% 100%;
	background-repeat: no-repeat;
	background-attachment: fixed;
}

Works a charm and all I do is add a new table.project(projectid) to the css for each board.

Anyway long story short, I think it would be nice to have it by design that users can upload their own background images.

Thanks

**Comment by BlueTeck** [comment]
I now that you want a bockgound image, but there are also 2 pull request for "official support" of custom css: #618 #666

**Comment by fguillot** [comment]
In the development branch, you have now a unique css class name for each board: `board-project-id` (by example `board-project-1`). You can also add some custom css from the settings page since I merged the pull-request #666.
